doc/topics/packaging/testing.rst

```diff
@@ -0,0 +1,117 @@
+.. _pkging-testing:
+
+================
+Testing Packages
+================
+
+The Package Test Suite
+======================
+
+The salt repo provides a test suite for testing basic functionality of our packages at ``<repo-root>/pkg/tests/``.
+The install, upgrade, and downgrade tests can be run locally and are run on most PRs that are submitted against Salt.
+
+
+.. warning::
+
+    These tests are destructive to your system, as they install the built packages onto the
+    system. They may also install older versions in the case of upgrades or downgrades.
+    It is recommended to run these in an isolated system, preferrably a VM.
+
+Setup
+=====
+In order to run the package tests, the `relenv <https://github.com/saltstack/relative-environment-for-python>`_
+onedir and built pacakges need to be placed in the correct locations.
+
+All salt packages for the wanted testing version must be placed in ``<repo-root>/pkg/artifacts/``,
+and the onedir must be located under ``<repo-root>/artifacts/``. Additionally, to ensure complete parity with Salt's CI/CD
+suite, the ``nox`` virtual environment can be placed at ``<repo-root>/.nox/test-pkgs-onedir`` The following are a few ways this can be accomplished easily.
```

Comment:
**Nitpick (style, non-blocking):** Passive voice again. The fix: "Additionally, to ensure complete parity with Salt's CI/CD suite, place the ``nox`` virtual environment in ..."

**Suggestion (organization, non-blocking):** How about we introduce the next sections this way?

You can ensure parity by installing the package test suite through a few possible methods:

- Using ``tools``
- Downloading individually
- Running the tests